ab3d2_source/c/zone.h

```diff
@@ -191,14 +191,34 @@ extern ZEdgePVSHeader** Lvl_ZEdgePVSHeaderPtrsPtr_l;
 /** Number of visible joining endges currently in view */
 extern WORD Zone_VisJoins_w;
 
-/** Array indexes of the end coordinates of the visible joining edge when Zone_VisJoins_w == 1 */
-//extern WORD Zone_EdgeClipIndexes_vw[2];
-
 /** Onscreen maximum clip extents - either the full screen, or the portion visible through the edge */
 extern WORD Draw_ZoneClipL_w;
 extern WORD Draw_ZoneClipR_w;
 
-void Zone_UpdateVectors(void);
 void Zone_CheckVisibleEdges(void);
 
+/**
+ * Enumeration of the possible crossings from one Zone to another via their shared edge.
+ */
+typedef enum {
```

Comment:
Be aware that GCC uses 32 bit storage to represent enums in memory.
There's a compiler switch '-fshort-enums' to make it use 16bit if possible.